### PR TITLE
#1646 ACTIONS - MA FIATer for MSA-GRH - deeming functionality minor bugs

### PR DESCRIPTION
**Pull request by C-Love**
Changed the dialog parameters for deeming income and updated the dialog used for the deeming members selection.

Additionally moved the testing_run to be true ONLY when deeming income was selected. We seem to have sufficient testing for the standard run. 

If you need a case to test on the error report from TE in the team email has one with 2 members that deeming functionality can be tested. 

Selected all team members for review, but I am looking for two reviews.

### CODE REVIEW

**Review by IlseFerris — CHANGES_REQUESTED** [review]
I selected two members, and got the message "only select one member", but then additional checkboxes for MEMB 01 and MEMB 02 got added to the HH_member_dialog, for a total of 4. Did it again and got 6 checkboxes, etc.

File: actions/ma-fiater-for-grh-msa.vbs [hunk 1]
```diff
@@ -747,6 +747,7 @@ Do
 Loop until are_we_passworded_out = FALSE														' }
 
 Call back_to_SELF
+Dim client_array
 
 DO
```
Comment (MDR22) [inline actions/ma-fiater-for-grh-msa.vbs:752]: Although you didn't change this, I kept running into a bug where if more than one person is selected, you get a msgbox about selecting one person only. When HH_member_custom_dialog(HH_member_array) is called again, it then adds the HH Membs to the dialog again (so if you have 2 HH members in the dialog to start, you will have 4 after the first msgbox, and then 6 again after it calls HH_member_custom_dialog(HH_member_array) again.

**Review by MDR22 — COMMENTED** [review]
I tried on the case from the team email and didn't run into any issues besides the one comment related to selecting too many HH members. Looks good!

**Comment by C-Love** [comment]
Ok - initial dialog updated - well blanking out of the client_array before calling the function to select the member to make sure we don't multiply all the hh members. Second review needed.

**Review by IlseFerris — APPROVED** [review]
I tested this on the case in the TE, and it worked as expected. I approve thee!

**Review by MDR22 — APPROVED** [review]
No issues with duplicating HH Membs now.

**Pull request merged**